homeassistant/components/wemo.py

```diff
@@ -91,6 +92,7 @@ def discovery_dispatch(service, discovery_info):
         if device is None:
             device = pywemo.discovery.device_from_description(url, None)
 
-        discovery_info = (device.name, device.model_name, url, device.mac)
+        discovery_info = (device.name, device.model_name, url, device.mac,
```

Comment:
Ah, I didn't realize there was this strong of linkage there.

I can put up a patch, but I still think one of you should test the local discovery path, so it might be easier for that person to do it. But happy to if it helps.